common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -37,7 +37,8 @@
         @defining(media.channelId.getOrElse("")){ channelId: String =>
             <div data-media-atom-id="@media.id" class="@RenderClasses(Map(
                 ("u-responsive-ratio", !isPaidFor),
-                ("u-responsive-ratio--hd", sixteenByNine),
+                ("u-responsive-ratio--hd", !verticalVideo),
```

Comment:
could this still be `sixteenByNine` rather than `!verticalVideo`?  I'm assuming we don't have a sixteenByNine attribute on vertical videos but maybe im wrong...

Reply:
I'm not that familiar with this syntax, but does sixteenByNine get the value true when playable is true? If that's the case, maybe it should be `("u-responsive-ratio--hd", sixteenByNine && !verticalVideo),` ?